fpmsyncd/fpmlink.cpp

```diff
@@ -18,6 +18,18 @@ void netlink_parse_rtattr(struct rtattr **tb, int max, struct rtattr *rta,
         {
             tb[rta->rta_type] = rta;
         }
+        else
+        {
+            /* Latest FRR is sending RTA_ENCAP with NLA_F_NESTED bit set*/
```

Comment:
Please rephrase the comment to mention FRR version or remove the comment itself. "Latest" may not be relevant sometime in future

Reply:
Sure, I will update the comment.